src/components/application_manager/src/commands/mobile/set_global_properties_request.cc

```diff
@@ -285,8 +284,10 @@ void SetGlobalPropertiesRequest::on_event(const event_engine::Event& event) {
   ApplicationSharedPtr application =
       application_manager_.application(connection_key());
 
-  SendResponse(
-      result, result_code, return_info, &(message[strings::msg_params]));
+  SendResponse(result,
+               result_code,
+               return_info.c_str(),
```

Comment:
@vlantonov, what will return c_str() in case empty string?? As I remember it returns "" instead of NULL. In this case command will send info="". It's a defect.